app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -71,17 +71,17 @@ $ske-course-video$
 
 They're usually online, but some courses can be classroom based and some can be a mixture of both. 
 
-Course lengths usually range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length is appropriate based on your gap in knowledge. Different providers may offer different course lengths.
+Course lengths usually range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.
 
-You can also do a part-time course and complete it over a longer period, or sometimes shorter. Your SKE provider will be able to discuss your options with you.
+You can do a part-time course and complete it over a longer period or complete it in a shorter time with compressed hours. Your SKE provider will be able to discuss your options with you.
 
 ## When to complete your course
 
 You should complete your course before starting your teacher training. If you run out of time, your provider may allow you to continue your course at the same time as your teacher training.
 
 ## Get an SKE bursary
 
-If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. How and when your bursary is paid will vary by your training provider.
+If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will affect your bursary amount. How and when your bursary is paid will vary by your training provider.
```

Comment:
```suggestion
If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will also affect your bursary payment. How and when your bursary is paid will vary by your training provider.
```
Is `payment` more appropriate than `amount`? From what I'm understanding, it is the amount you get per week that is affected rather than the overall amount if you do compressed hours (but I might be wrong). Something to clarify with the SKE team.

Reply:
would probably help to spell out to users exactly HOW it is affected